harness/tests/test_horovod.py

```diff
@@ -6,7 +6,7 @@
 @pytest.mark.parametrize("debug", [True, False])
 @pytest.mark.parametrize("auto_tune", [True, False])
 @pytest.mark.parametrize("tensor_fusion_threshold", [64, 128, 512])
-@pytest.mark.parametrize("tensor_fusion_cycle_time", [5, 20])
+@pytest.mark.parametrize("tensor_fusion_cycle_time", [1, 5, 20])
```

Comment:
maybe cut the number of tests a little bit, trying one default and one non-default value seems sufficient:

```suggestion
@pytest.mark.parametrize("tensor_fusion_cycle_time", [1, 5])
```